event-code/README.md

```diff
@@ -0,0 +1,10 @@
+# Event Code
+
+```
+{
+	id: string,
+	code: string,
+	created_at: <timestamp>,
+	updated_at: <timestamp>
+}
```

Comment:
Can you please check once I think we used to add `role` as well

Reply:
Done